packages/editor/src/components/assets/SceneAssetsPanel.tsx

```diff
@@ -251,20 +251,26 @@ const SceneAssetsPanel = () => {
         </div>
       )
     }
-    if (searchedStaticResources.value)
-      if (selectedCategory.value)
-        return (
-          <>
-            {searchedStaticResources
-              .filter((resource) => {
+    if (searchedStaticResources.value) {
+      return (
+        <>
+          {searchedStaticResources
+            .filter((resource) => {
+              if (selectedCategory.value !== null) {
```

Comment:
This case will be true for empty string as well.
We can do this:  `if (selectedCategory.value?.name && selectCategory.value?.object)`

Reply:
Done!